microsoft-edge/accessibility/build/aria-and-ui-automation.md

```diff
@@ -11,9 +11,9 @@ keywords: accessibility, accessibility for developers, accessible websites, edge
 ms.custom: seodec18
 ---
 
-# ARIA and UI Automation in Microsoft Edge
+# ARIA and UI automation in Microsoft Edge
 
-The W3C defines Accessible Rich Internet Applications (ARIA) as a syntax for making dynamic web content and custom UI accessible. Microsoft Edge recognizes the ARIA role, state, and property information and exposes it to assistive technologies, which in turn can use the [Microsoft UI Automation](https://blogs.msdn.microsoft.com/winuiautomation/) APIs to retrieve the information.
+The W3C defines Accessible Rich Internet Applications (ARIA) as a syntax for making dynamic web content and custom UI accessible. Microsoft Edge recognizes the ARIA role, state, and property information and exposes it to assistive technologies, which in turn can use the [Microsoft UI Automation](https://blogs.msdn.microsoft.com/winuiautomation/) APIs to retrieve the information. 
 
 Visit [HTML5Accessibility](https://html5accessibility.com) for information on which new HTML5 features are accessibly supported by Microsoft Edge.
 
```

Comment:
Nitpick: every bulleted list has a period at the end of the last line, but not at the end of the first line. The first lines are not complete sentences, but I thought we were supposed to match the punctuation throughout the list. As in: if one bulleted item is a full sentence and has a period then all items get a period.

Reply:
Previously discussed w/ mgmt, discussed MWSG & its assumed context re: periods in list items.  Other than list of, say, function names, generally, list items need periods.